src/components/picker/option/index.tsx

```diff
@@ -34,8 +34,6 @@ export class SmoothlyPickerOption {
 	}
 	componentWillLoad() {
 		this.name = this.name ?? this.value
-	}
-	componentDidLoad() {
```

Comment:
the picker relies on some of this to happen post loading cause some element references (slotted children) does not exist until after load. preselected elements showing up in the element depended on this. need another solution. I'm going to look into other issues in the picker so i could try to fix everything in the picker.

Reply:
for some visuals
picker loaded with these changes
![image](https://github.com/utily/smoothly/assets/25643315/da215ff3-81b6-4696-b60b-5014a05be767)
Picker loaded without these changes
![image](https://github.com/utily/smoothly/assets/25643315/11c61597-82c4-499e-9645-6ed9ca5bbd9a)